### DISCUSSION TITLE
lmdeploy是否支持qwen2的工具调用？

### DISCUSSION THREAD

**Discussion by lai-serena** [InternLM/lmdeploy#2220] [Q&A] [answered]
https://lmdeploy.readthedocs.io/zh-cn/latest/serving/api_server_tools.html
这里写着“LMDeploy 支持 InternLM2, InternLM2.5 和 Llama3.1 模型的工具调用。”是否意味着不支持Qwen2的工具调用？

**Comment by lvhan028** [comment]
恩，没有支持 Qwen2 的工具调用

**Reply by lai-serena** [reply]
后续会考虑支持Qwen2的工具调用吗？

**Reply by lvhan028** [reply, marked as answer]
暂时不考虑。
